### PR TITLE
Updated the UI references

### PR DESCRIPTION
**Pull request by mprintz**
Added the new UI pictures and references to the button locations. Also added more info on the retention policy.

Closes #

Added the new UI pictures and references to the button locations. Also added more info on the retention policy.

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

**Comment by mprintz** [comment]
My pictures pasted too large in this request. Sorry. The brackets are showing too I think. Let me know if you want better pictures or if you have a way to resize them correctly.

File: content/influxdb/cloud/organizations/buckets/update-bucket.md [hunk 1]
```diff
@@ -23,24 +23,25 @@ If you change a bucket name, be sure to update the bucket in the above places as
 
 ## Update a bucket's name in the InfluxDB UI
 
-1. In the navigation menu on the left, select **Data (Load Data)** > **Buckets**.
+1. In the navigation menu on the left, select **Data ** > **Buckets**.
 
-    {{< nav-icon "data" >}}
+    {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:28]: This is a convention we've established to address both the collapsed and expanded states of the left nav. These should not be changed.

File: content/influxdb/cloud/organizations/buckets/update-bucket.md [hunk 2]
```diff
@@ -23,24 +23,25 @@ If you change a bucket name, be sure to update the bucket in the above places as
...
 
-2. Click **Settings** under the bucket you want to rename.
+2. Click **Settings** to the right of the current name of the bucket that you want to rename.
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:30]:
```suggestion
2. Click **Settings** to the right of the bucket you want to rename.
```

File: content/influxdb/cloud/organizations/buckets/update-bucket.md [hunk 3]
```diff
@@ -23,24 +23,25 @@ If you change a bucket name, be sure to update the bucket in the above places as
...
 
 ## Update a bucket's retention policy in the InfluxDB UI
 
-1. In the navigation menu on the left, select **Data (Load Data)** > **Buckets**.
-
-    {{< nav-icon "data" >}}
+1. In the navigation menu on the left, select **Data ** > **Buckets**.
 
+    {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}
+
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:40]: This is a convention we've established to address both the collapsed and expanded states of the left nav. These should not be changed.

File: content/influxdb/cloud/organizations/buckets/update-bucket.md [hunk 4]
```diff
@@ -23,24 +23,25 @@ If you change a bucket name, be sure to update the bucket in the above places as
...
-    {{< nav-icon "data" >}}
+1. In the navigation menu on the left, select **Data ** > **Buckets**.
 
+    {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}
+    
 2. Click **Settings** next to the bucket you want to update.
-3. In the window that appears, edit the bucket's retention policy.
-4. Click **Save Changes**.
+3. In the window that appears, select from the predefined time ranges for the bucket's retention policy. Note: You can use the CLI commands further below on this page to set the retention policy to different values not shown in the UI.
+    {{![image](https://user-images.githubusercontent.com/9125473/109828711-7fbfff00-7c02-11eb-94fa-9acc07a65296.png)}}
```
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:43]:
```suggestion
3. In the window that appears, under **Delete data**, select a retention period:

    - **Never**: data in the bucket is retained indefinitely.
    - **Older Than**: select a predefined retention period from the dropdown menu.
    
    {{% note %}}
Use the [`influx bucket update` command](#update-a-buckets-retention-policy) to set a custom retention policy.
    {{% /note %}}
```

**Review by sanderson — CHANGES_REQUESTED** [review]
- The menu references and nav icos follow a convention we've established to address both the collapsed and expanded states of the left nav. 
- Images should be added directly to the repo rather than served from Github. In this case, I don't think the images are necessary and have suggested changes to help clarify the instructions.

**Comment by mprintz** [comment]
Thanks @sanderson !

**Review by mprintz — COMMENTED** [review]
Looks good. Thanks @sanderson

File: content/influxdb/cloud/organizations/buckets/update-bucket.md (hunk 3 above)
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:40]:
```suggestion
1. In the navigation menu on the left, select **Data (Load Data)** > **Buckets**.

    {{< nav-icon "data" >}}
    
```

File: content/influxdb/cloud/organizations/buckets/update-bucket.md (hunk 1 above)
Comment (sanderson) [inline content/influxdb/cloud/organizations/buckets/update-bucket.md:28]:
```suggestion
    {{< nav-icon "data" >}}
```

**Review by sanderson — APPROVED** [review]

**Pull request merged**